examples/erosion2/storage.py

```diff
@@ -1,33 +1,64 @@
 import shelve
+import string
+import time
+
+
+_CHARS = sorted(string.ascii_lowercase + string.digits)
+_N_CHARS = len(_CHARS)
+
+
+def _encode_id(num):
+    alias = ""
+    while num:
+        alias = _CHARS[num % _N_CHARS] + alias
+        num //= len(_CHARS)
```

Comment:
better hope `num` is never negative ;)

Reply:
I've noticed that the original function runs `abs` on the input. My reasoning for removing that check was that `num` couldn't be negative in this app (I may be overlooking something). I've also renamed the function to start with an underscore to indicate internal use. But I could have added the one line and be on the safe side, in much less time than it took me to write this paragraph :)